File: Week 6/Problem Set 6/sentimental-mario-less/mario.py
# TODO
from cs50 import get_int

# Proper usage
while True:
    height = get_int("Height: ")
    if height <= 8 and height > 0:
        break

# Printing the pyramid
for i in range(height):
    print(" " * (height - 1 - i), end="")
    print("#" * (i + 1))

File: Week 1/Problem Set 1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>
int main(void)
{
    int H;
    do
    {
        H = get_int("Height: ");
    }
    while (H < 1 || H > 8);
    for (int n = 0; n < H; n++)
    {
        for (int w = H - 1; w > n; w--)
        {
            printf(" ");
        }
        for (int z = -1; z < n; z++)
        {
            printf("#");
        }
        for (int y = 0; y < 2; y++)
        {
            printf(" ");
        }
        for (int k = -1; k < n; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}